### PR TITLE
Formatting id to class and spacing

### PR DESCRIPTION
**Pull request by Nicolebt28**
-fixed id to class
-fixed indenting spaces

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,24 +1,40 @@
...
+    <li><a href="#projects_section">Projects</a></li>
+    <li><a href="#contact">Contact</a></li>
+   <nav>
+</header>
     
- <main id="home">
-    <h1 id= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
-    <h2 id="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
+<main id="home">
+  <h1 class= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 2]
```diff
@@ -1,24 +1,40 @@
...
+  <h2 class="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
 </main>
     
-<div id= "projects_section">
+<div id= "projects_section" class= "projects_section">
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 3]
```diff
@@ -1,24 +1,40 @@
...
   
-    <h2 id="project_opener"> Here are some sites I've created...</h2>
+  <h2 class="project_opener"> Here are some sites I've created...</h2>
   
-    <p id= "project_text" class= "project_opener">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic ...</p>
-    </div>
\ No newline at end of file
+  <p class= "project_text" class= "project_opener">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic ...</p>
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 4]
```diff
@@ -1,24 +1,40 @@
...
+  
+  <h2 class= "checklist"> To-Do List</h2>
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 5]
```diff
@@ -1,24 +1,40 @@
...
+  <img class="projectpic" src="https://images.unsplash.com/photo-1547480053-7d174f67b557?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Check list on a desk">
+       
+  <h2 class= "recipe">Cooky Cookie's Recipe Card</h2>
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 6]
```diff
@@ -1,24 +1,40 @@
...
+  <img class="projectpic" src="https://images.unsplash.com/photo-1512058454905-6b841e7ad132?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1586&q=80" alt="Recipe on counter">
+    
+  <h2 class= "timer"> Tabata Timer</h2>
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 7]
```diff
@@ -1,24 +1,40 @@
...
+  <img class="projectpic" src="https://images.unsplash.com/photo-1431499012454-31a9601150c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Timer in hand">
+</div>
+    
+<div id= "contact" class= "contact">
```
Comment (cory-tanner) [inline index.html]: 🔥extra space `class= "foo"`

File: index.html [hunk 8]
```diff
@@ -1,24 +1,40 @@
...
+  <h2>Email: email@web.com</h2>
+  <h2>Phone: 888-888-8888</h2>
+  <h2>Web: nicoletanner.me</h2>
+</div>
+
```
Comment (cory-tanner) [inline index.html]: 🔥 Extra lines

File: style.css [hunk 9]
```diff
@@ -1,66 +1,95 @@
...
-    width: 100%;
-    text-align: center;
-  
-  }
\ No newline at end of file
+  font-family: 'Josefin Sans', sans-serif;
+  box-sizing: border-box;
+}
+
+main {
```
Comment (cory-tanner) [inline style.css]: Apply this with a class, avoid styling HTML elements via the element name at all costs

File: style.css [hunk 10]
```diff
@@ -1,66 +1,95 @@
...
+  color:#b2bec3;
+}
+
+.welcome_subtitle{
+  font-size: 40px;
+  color: #b2bec3;
+  width: 75%;
+}
+
+nav{
```
Comment (cory-tanner) [inline style.css]: Same as what I said above

File: style.css [hunk 11]
```diff
@@ -1,66 +1,95 @@
...
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  background-color: hsla(198, 12%, 73%, 0.25);
+}
+
+li {
```
Comment (cory-tanner) [inline style.css:34]: Same as what I said above

File: style.css [hunk 12]
```diff
@@ -1,66 +1,95 @@
...
+  list-style-type: none;
+  font-size: 30px;
+  color: #218c74;
+  float: left;
+  display: float;
+  padding: 15px;
+}
+
+a {
```
Comment (cory-tanner) [inline style.css:43]: Same as what I said above

File: style.css [hunk 13]
```diff
@@ -1,66 +1,95 @@
...
+  font-size: 30px;
+  color: #218c74;
+  text-decoration: none;
+}
+
+a.visited{
```
Comment (cory-tanner) [inline style.css]: Same as what I said above

File: style.css [hunk 14]
```diff
@@ -1,66 +1,95 @@
...
+  font-size: 30px;
+  color: #218c74;
+}
+
+.projects_section{
```
Comment (cory-tanner) [inline style.css]: Add a space between the class name and `{`

File: style.css [hunk 15]
```diff
@@ -1,66 +1,95 @@
...
+  padding: 40px;
+}
+
+.project_opener {
+  font-size: 35px;
+  color: #b33939;
```
Comment (cory-tanner) [inline style.css:60]: Apply colors with CSS custom properties. `var(--foo-color)`. https://developer.mozilla.org/en-US/docs/Web/CSS/--*

File: index.html [hunk 16]
```diff
@@ -1,24 +1,40 @@
...
+    <li><a href="#home">Home</a></li>
+    <li><a href="#projects_section">Projects</a></li>
+    <li><a href="#contact">Contact</a></li>
+   <nav>
+</header>
     
- <main id="home">
-    <h1 id= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
-    <h2 id="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
+<main id="home">
```
Comment (cory-tanner) [inline index.html:14]: Has an ID :(

File: index.html [hunk 17]
```diff
@@ -1,24 +1,40 @@
...
+  <h1 class= "welcome"> Hello! My name is Nicole, an aspiring web deveolper and web designer.</h1>
+  <h2 class="welcome_subtitle"> Welcome to my portfolio, feel free to shuffle through and see what you like!</h2>
 </main>
     
-<div id= "projects_section">
+<div id= "projects_section" class= "projects_section">
   
-    <h2 id="project_opener"> Here are some sites I've created...</h2>
+  <h2 class="project_opener"> Here are some sites I've created...</h2>
```
Comment (cory-tanner) [inline index.html:21]: use CSS ellipsis https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/

File: index.html (hunk 3 above)
Comment (cory-tanner) [inline index.html]: You have 2 classes?

File: index.html [hunk 18]
```diff
@@ -1,24 +1,40 @@
...
+       
+  <h2 class= "recipe">Cooky Cookie's Recipe Card</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1512058454905-6b841e7ad132?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1586&q=80" alt="Recipe on counter">
+    
+  <h2 class= "timer"> Tabata Timer</h2>
+  <img class="projectpic" src="https://images.unsplash.com/photo-1431499012454-31a9601150c9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1650&q=80" alt="Timer in hand">
+</div>
+    
+<div id= "contact" class= "contact">
+  <h2>Email: email@web.com</h2>
```
Comment (cory-tanner) [inline index.html]: Email uses an email link

File: index.html [hunk 19]
```diff
@@ -1,24 +1,40 @@
...
+  <h2>Phone: 888-888-8888</h2>
```
Comment (cory-tanner) [inline index.html]: leave your phone number off the site. but also use a tele link instead of a heading

**Review by cory-tanner — CHANGES_REQUESTED** [review]

**Comment by cory-tanner** [comment]
Next step is to fix these changes and push a second commit to the branch. That commit will show here in this PR @Nicolebt28

**Comment by cory-tanner** [comment]
Still has updates that need to be made relative to using classes to apply styling.

**Comment by Nicolebt28** [comment]
can you give me an example of 1 or 2 elements you would change to use
classes in the way you're describing?

**Pull request closed without merging**